test/action.yml

```diff
@@ -46,12 +46,24 @@ runs:
       run: |
         go test -p 1 -timeout 30m -v ./suites/${{inputs.name}}
 
-    # Print snap logs
-    - if: always()
+    # Check file existence
+    - id: check_files
+      uses: andstor/file-existence-action@v1
+      with:
+        files: "${{github.action_path}}/suites/${{inputs.name}}/snap.log"
+
+    # Print snap logs if the file exists
+    - if: steps.check_files.outputs.files_exists == 'true'
       shell: bash
       working-directory: ${{github.action_path}}
       run: |
         echo "::group::Snap Logs"
         cat ./suites/${{inputs.name}}/snap.log
         echo "::endgroup::"
+
+    # Print info logs if the file does not exist
+    - if: steps.check_files.outputs.files_exists == 'false'
+      shell: bash
+      run: |
+        echo "snap logs are not available because this snap has no service"
```

Comment:
How about:
`if: always() && inputs.print_logs != 'false'` and the original step?

Reply:
The `print_logs` input should not be required and should default to true. CLI will set it to false.